CHANGELOG.md

```diff
@@ -18,6 +18,10 @@ release.
 - Rename metrics `jvm.memory.usage` to `jvm.memory.used` and `jvm.memory.usage_after_last_gc`
   to `jvm.memory.used_after_last_gc`
   ([#536](https://github.com/open-telemetry/semantic-conventions/pull/536))
+- BREAKING: Remove `event.domain` from log event attributes.
```

Comment:
Nit: This could be combined into a single changelog entry. 

> - BREAKING: Remove `event.domain` from log event attributes and adapt `event.name` definition to include `namespace`